articles/server-apis/rails.md

```diff
@@ -23,19 +23,16 @@ tags:
 
 **Otherwise, Please follow the steps below to configure your existing Ruby on Rails app to use it with Auth0.**
 
-### 1. Add jwt dependency to your Gemfile
+### 1. Install the knock gem
 
-You need to add the jwt dependency.
-
-Open your Gemfile and add the following:
-
-```bash
-gem 'jwt'
-```
+Setup knock in 2 minutes: 
+<a href="https://github.com/nsarno/knock#getting-started" target="_blank">
```

Comment:
I know @hzalaz just reviewed the code, but we need to put all of the steps on what to configure here, not a link to your Github in this case. People will still know it's your lib since we're installing it :D